### ISSUE TITLE
Performance drop for intel uhd graphics 620 and dxvk with lutris 0.5.7.1

### ISSUE THREAD

**Issue by Feichtmeier** [lutris/dxvk#1]
Ubuntu 20.04
Lutris 0.5.7.1
UHD Graphics 620 (Wiskey Lake)
16 RAM
World of Warcraft usind DXVK+VK3D DXVK VErsion 1.7L-3322da4-2

After using Lutris 0.5.7.0 previously I noticed a FPS drop of 5-10 frames after installing 0.5.7.1
One might say that you should not play WoW with intel hd graphics anyways but well, it's playable with DXVK :) So 5-10 FPS is something you really notice.
Anything changed here that changes DXVK with intel ?
Best regards

**Comment by Feichtmeier** [comment]
Thanks @tannisroot and sorry for opening this in the wrong repo!
It's directx11, i have only the options for this and directx11legacy.
Lowering the dxvk version made it much worse so I switched back to 1.7L-3322da4-2

What made it better was upgrading my graphics stack from the oibaf ppa and I can use the 0.5.7.1 version without fps loss. 
Previously I had the graphics stack that comes preinstalled with Ubuntu 20.04.0

**Comment by ghost** [comment]
You might want to try DXVK 1.7.1 version.

**Comment by Feichtmeier** [comment]
Are the DXVK versions automatically downloaded and appear in the Lutris list after a new release has been officially been tagged on lutris/dxvk ?
![grafik](https://user-images.githubusercontent.com/15329494/90235329-6bacdf80-de21-11ea-8abd-7715c67156c2.png)

**Comment by tannisroot** [comment]
Could you try switching Wine version to 5.7-8? Thanks.

**Comment by Feichtmeier** [comment]
@tannisroot - np, here you go:
lutrix 5.7.8 + DXVK 1.7L-9f6f6a7 - 22 fps

wine version lutrix 5.6.5 + DXVK 1.7L-56fe52c - 33 fps
(same position ofc - in front of the timewalking wizard in dalaran in front of that big blue portal)

Are lutris wine versions and lutris DXVK versions automatically downloaded?

**Comment by ghost** [comment]
> @tannisroot - np, here you go:
> lutrix 5.7.8 + DXVK 1.7L-9f6f6a7 - 22 fps
> 
> wine version lutrix 5.6.5 + DXVK 1.7L-56fe52c - 33 fps
> (same position ofc - in front of the timewalking wizard in dalaran in front of that big blue portal)
> 
> Are lutris wine versions and lutris DXVK versions automatically downloaded?

Type 1.7.1 in "DXVK version" and it should get it automatically.

**Comment by Feichtmeier** [comment]
This does not work with my setup:
![grafik](https://user-images.githubusercontent.com/15329494/90243350-0c55cc00-de2f-11ea-848e-82410622a498.png)
Sorry for the noise :(

**Comment by ghost** [comment]
> This does not work with my setup:
> ![grafik](https://user-images.githubusercontent.com/15329494/90243350-0c55cc00-de2f-11ea-848e-82410622a498.png)
> Sorry for the noise :(

Yeah can confirm I can't get it either, very strange can get earlier 1.7 version (maybe they changed something?), better then wait for 1.7.1L build.

**Comment by ghost** [comment]
If any have problem to update DXVK version on lutris, delete all files inside "dxvk" folder on:

`/home/x/.local/share/lutris/runtime/dxvk`

And start lutris again, it should refresh with latest DXVK version (which is now v1.8.1L)
